### ISSUE TITLE
Restore AR Toolkit to v200

### ISSUE THREAD

**Issue by nCastle1** [Esri/arcgis-toolkit-dotnet#467]
AR Toolkit will not be included in the v200 release due to issues with AR Core support. See Microsoft's bug for details: https://github.com/dotnet/maui/issues/10152

Once the ARCore dependency issue is resolved, we will be able to proceed with testing and release for Android.

**Comment by SokoFromNZ** [comment]
When can we expect the Toolkit to work with MAUI? I would only need Android support at the moment...

**Comment by dotMorten** [comment]
@SokoFromNZ The ball is in Microsoft's court, since their ARCore bindings are broken for .NET 6+. Until they fix that, we can't really bring it back. The main branch does have a working iOS implementation (but I get that doesn't help you).

**Comment by SokoFromNZ** [comment]
Yeah... MAUI is really horrible. I've opened Issues there as well and none of them are taken care of at the moment.
I still cannot understand how they dared to release such a buggy framework.
Anyhow...thanks for the feedback.
I kinda missed the the link to the issue in OP is to the MAUI project.

**Comment by dotMorten** [comment]
🤷‍♂️ I've generally been very happy with the quality of .NET MAUI.

**Comment by SokoFromNZ** [comment]
Hmmm... interesting. You are the first one I know of.
I'm mainly developing with .NET and WPF since 15 years now and compared to this, MAUI feels like a beta version of a framework.
We dared to start a new big project with MAUI in May which should be released in January. We had to cut back from releasing it for Win, iOS & Android to just Android, and it's still a hassle. 70% of my time I devote to find workarounds to MAUI bugs.

**Comment by mbranscomb** [comment]
@SokoFromNZ I'd love to hear more about how you use our AR Toolkit today and/or any future plans for implementing AR. Please feel free to email me: MBranscomb@esri.com.

Thanks

**Comment by SokoFromNZ** [comment]
@mbranscomb I wanted to use the BasemapGallery in our MAUI app. But ended up implementing a basemap chooser by myself as this does not work.
To be frank: I use the BasemapGallery in a WPF app but today I'm removing it there as well. Reasons:
- No localization support (Need the descriptions & tooltips in different languages)
- I could not figure out how to save & restore the last selection of the user after restart of the app

**Comment by dotMorten** [comment]
@grimmi We haven't worked on the AR bits in a while so things might have changed from out under us, since the project is currently on hold due to the Android bits has been deprecated by both Google and Microsoft. When we did work on it iOS was working. There is a work-in-progress branch that might fix some of your issues here: https://github.com/Esri/arcgis-maps-sdk-dotnet-toolkit/pull/466
